cmd/convert.go

```diff
@@ -13,12 +13,11 @@ import (
 )
 
 const (
-	convertArgMin = 1
-	convertArgMax = 1
+	convertArgMax         = 1
+	defaultOutputFileName = "conf"
```

Comment:
Shouldn't this be called `defaultOutputExt` or something?

Reply:
we talked about this in a huddle already

Reply:
changed to "config". It's the default output file name.